index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Landing Page</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Raleway"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"
+      integrity="sha512-iBBXm8fW90+nuLcSKlbmrPcLa0OT92xO1BIsZ+ywDWZCvqsWgccV3gFoRBv0z+8dLJgyAHIhR35VZc2oM/gI1w=="
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <button class="btn"><i class="fas fa-bars fa-2x"></i></button>
+    <div id="container">
+      <div class="heading-primary">
+        <h1>Dr. Norman Borlaug</h1>
+        <p>The man who saved a billion lives</p>
+      </div>
+      <div id="wrapper">
+        <img src="img/10613180113_fdf7bcd316_b.jpg" alt="" />
```

Comment:
You didn't add the alt text to the img tag. Adding the alt text to the img tag improves SEO and also makes the page more accessible.